WTWSMS/history/titles/c_salamanca.txt

```diff
@@ -10,16 +10,19 @@
 	liege = "k_visigoths"
 }
 480.1.1={
-	holder = 1301295 # Salla, Visigothic dux in Spain 483
+	holder=1301295
```

Comment:
Comments make reading the title history code much simpler.

Reply:
I need to recheck plenty of the code because I don´t remember the reason of some deletions, I´ll get back to you later.